packages/amplify-velocity-template/tests/runner/mocha.js

```diff
@@ -1788,38 +1788,6 @@
     var JSONCov = require('./json-cov'),
       fs = require('browser/fs');
 
-    /**
-     * Expose `HTMLCov`.
-     */
-
-    exports = module.exports = HTMLCov;
```

Comment:
Is this not referenced somewhere?

Reply:
I don't believe we are using mocha for in browser testing as this is being used in node environment.